src/layouts/base-new/index.tsx

```diff
@@ -26,6 +28,17 @@ const { ConsentManager, openConsentManager } = createConsentManager({
 	preset: 'oss',
 })
 
+// TODO find better place for this
+const shouldShowAlertBanner = true
```

Comment:
put this here to get product / designs eyes on it. but it should live elsewhere.... `data` , `content`, or `config`? 🤔

Reply:
I put it in `data`